src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -652,8 +652,7 @@ legend.schemaform-label.schemaform-file-label {
   border-bottom: 3px solid $color-primary;
 }
 
-.schemaform-downtime-after-steps {
-  margin-top: 0;
+.downtime-alert {
```

Comment:
I didn't style the `va-alert` selector directly just in case some other `<va-alert>` ended up on the page that would look odd with extra margin :shrug: